common/common-profile.h

```diff
@@ -0,0 +1,44 @@
+/*
+
+Copyright (c) 2025, MithicSpirit
```

Comment:
As above, please change this line to "Copyright (c) 2025, the GameMode contributors".